lib/facter/winlogbeat_version.rb

```diff
@@ -0,0 +1,10 @@
+require 'facter'
+Facter.add('winlogbeat_version') do
+  confine :kernel => %w(Windows) # rubocop:disable Style/HashSyntax
+  if File.exist?('c:\Program Files\Winlogbeat\winlogbeat.exe')
```

Comment:
I haven't learned this part of puppet very well yet but can this file read in the install_dir as a variable?